openvpn/session/authentication.go

```diff
@@ -0,0 +1,47 @@
+package session
+
+import (
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/openvpn/middlewares/client/auth"
+	"github.com/mysterium/node/session"
+)
+
+const sessionSignaturePrefix = "SessionId:"
```

Comment:
IMHO too generic, I would use word 'Myst' somewhere.

Reply:
-> "MystVpnSessionId:". It's just for signing/validating session/signature anyway.